test/end-to-end.js

```diff
@@ -1227,3 +1229,15 @@ function bootSandboxWithOptions(options, done) {
     done(err, app);
   });
 }
+
+function localInstall(cwd, cb) {
+  var options = {
+    cwd: cwd,
+  };
+  var script = 'npm install';
+  return exec(script, options, function(err, stdout, stderr) {
```

Comment:
Please `debug` log the script and the options or at `cwd` before calling `exec`, e.g.

```js
debug('Running `%s` in %s', script, cwd);
exec(...);
```